generatedTypes/components/touchableOpacity/index.d.ts

```diff
@@ -30,7 +30,7 @@ export interface TouchableOpacityProps extends Omit<RNTouchableOpacityProps, 'st
      */
     customValue?: any;
     style?: StyleProp<ViewStyle> | Animated.AnimatedProps<StyleProp<ViewStyle>>;
-    onPress?: (props: TouchableOpacityProps | any) => void;
+    onPress?: (props?: TouchableOpacityProps | any) => void;
```

Comment:
What the reason we need `| any` here?

Reply:
Hmm good point. 
It was here before, but it might be irrelevant.. 
ill try to remove it and see if it affect us

Reply:
Ok, there was a reason for the `any` type.
In TouchableOpacity we allow to pass a prop called `customValue` which can be anything you want to pass and get back in the `onPress` callback. 
Because `customValue` can be anything, we have to support `any` type in the onPress callback, cause we can't tell what it'll be

Reply:
Yeah I see now. make sense :)